plugins/sqlmap.rb

```diff
@@ -28,30 +28,29 @@ def commands
 
       def cmd_sqlmap_connect(*args)
         if args.length == 0
-          print_error('Need a host, and optionally a port')
+          print_error("Need a host, and optionally a port")
```

Comment:
"Prefer single-quoted strings when you don't need string interpolation or special symbols..."
[Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)